### ISSUE TITLE
Image Appearing At End Of Posts

### ISSUE THREAD

**Issue by MrSaxon** [haveachin/reddit-bot#33]
This image appears at the end of every Reddit post without a video. It's usually preceeded by a swirling ring, as though the bot is trying to load something here but fails.

![reddit bot](https://github.com/haveachin/reddit-bot/assets/139659506/c6eefea3-4529-4044-b039-7aa568e64f6a)

Not sure what it is but is there a way of stopping it? Makes most posts look really untidy.

**Comment by haveachin** [comment]
Yeah, I also noticed that. I've also toyed with the idea of rewriting the bot to support multiple platforms like Instagram and Twitter, because it's also really starting to annoy me
